src/index.js

```diff
@@ -1,64 +1,67 @@
+/* eslint-disable import/no-cycle */
 import './style.css';
+import NewTodo from '../modules/crud.js';
+import createTodo from '../modules/create.js';
 
-const todoUl = document.getElementById('todoList');
+const form = document.getElementById('form');
+export const todo = document.querySelector('.add_todo');
+export const todoUl = document.getElementById('todoList');
+export const todoListss = JSON.parse(localStorage.getItem('todos')) || [];
+// const deleteButton = document.getElementById('clear_button');
 
-const todoLists = [
-  {
-    id: 1,
-    description: 'Connect to microverse morning call',
-    completed: true,
-  },
-  {
-    id: 2,
-    description: 'connect to microverse stand up call',
-    completed: false,
-  },
-  {
-    id: 3,
-    description: 'Prepare breakfast',
-    completed: false,
-  },
-  {
-    id: 4,
-    description: 'watch a youtube video on webpack',
-    completed: true,
-  },
-  {
-    id: 5,
-    description: 'have my bath',
-    completed: true,
-  },
-];
+// const todoLists = [
+//   {
+//     id: 1,
+//     description: 'Connect to microverse morning call',
+//     completed: true,
+//   },
+//   {
+//     id: 2,
+//     description: 'connect to microverse stand up call',
+//     completed: false,
+//   },
+//   {
+//     id: 3,
+//     description: 'Prepare breakfast',
+//     completed: false,
+//   },
+//   {
+//     id: 4,
+//     description: 'watch a youtube video on webpack',
+//     completed: true,
+//   },
+//   {
+//     id: 5,
+//     description: 'have my bath',
+//     completed: true,
+//   },
+// ];
```

Comment:
- [ ] Kindly remove commented-out code. They can make your code messy and more confusing for other developers and don't add functionality.